packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -676,6 +676,7 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
                 <oui-checkbox-description>
                     <span
                         data-translate="pci_projects_project_instances_add_localPrivateNetwork_add_public_network_info"
+                        data-translate-values="{ price: $ctrl.getPublicIpPriceInformation() || 0 }"
```

Comment:
**question:** Are you sure to display price as 0? Because if we display 0 in the Manager and them later charge the customer, that can be a legal issue? Do we have the approval from PU or PM to display "0"?

Reply:
Hello @anooparveti, With respect to legal issue I will check with them once they are back online. Will see if the possibility of null also exists or not. If it not we will remove the display of 0.

Reply:
PU confirmed me that price never be null. Hence I will remove the check.